cube.py

```diff
@@ -0,0 +1,37 @@
+"""
+This module defines a Cube class, and prints a list of 25
+randomly chosen faces of the cube.
+"""
+
+import random
+
+
+# Number of moves in scramble algorithm.
+K = 25
+
+
+""" A list of possible modifications on a move. To be appended to a
+face value like so: 'U2'.
+"""
+MODS = ["2", "'"]
```

Comment:
`""` is also a mod. Maybe you should start with a number of clockwise turns 1,2,3 and then figure out the front end representation later. You could even have a turn object that has a face and a number. Cube could just return an list of those.

Reply:
Is there a reason to use a 3 instead of a prime symbol? Will "3" be turned into a prime (') symbol with front end stuff?
Would a "turn" object would be a subclass of Cube?

Reply:
You basically want to separate any UI from any data. This class should just return manageable data. You are excited about the UI and are trying to get there early. Every step of a scramble should be a face and a number of turns. You could absolutely have a class for a scramble step, but `Cube` would have more of a "has a" relationship to it than an "is a" relationship, so it doesn't feel like a subclass to me.

You should check out enums. They are handy for stuff like this where there are a finite number of faces and ways to turn them: https://docs.python.org/3/library/enum.html#creating-an-enum